calour/transforming.py

```diff
@@ -33,16 +31,58 @@ def normalize(exp, reads=10000, axis=1, inplace=False):
 
     Returns
     -------
-    newexp : Experiment
+    ``Experiment``
         the normalized experiment
     '''
     if not inplace:
         exp = deepcopy(exp)
-    exp.data = preprocessing.normalize(exp.data, 'l1', axis=axis, copy=False) * reads
+    exp.data = preprocessing.normalize(exp.data, norm='l1', axis=axis) * total
+    return exp
+
+
+def scale(exp, axis=1, inplace=False):
+    '''Standardize a dataset along an axis
+
+    .. warning:: It will convert the sparse matrix to dense array.
+    '''
+    if not inplace:
+        exp = deepcopy(exp)
+    if exp.sparse:
+        exp.sparse = False
```

Comment:
we renamed it to _sparse?
anyway, we can get of it...using the exp.sparse property